DEVELOPING.md

```diff
@@ -149,3 +142,39 @@ This should create a build job, and then dispatch it to the build worker.
 You can view the build progress in the web UI or by viewing `/hab/svc/builder-worker/data/876066265100378112/log_pipe-876066265100378112.log`. Replace `876066265100378112` with the group ID output by the `start` command.
 
 Note: you will need to upload additional packages to the core origin for the `core/nginx` build to succeed. Follow the same procedure as for `core/hab-backline`. Currently `core/gcc` and `core/libedit` are required.
+
+### Setting up a package upstream
+
+It is possible to configure the on-premise builder to point to the hosted Builder site as an 'upstream'. This allows new packages from the upstream to get created in the on-premise instance automatically.
+
+In order to do so, create a file called `upstream.toml` with the following content:
+```
+[api]
+features_enabled = "upstream"
+
+[upstream]
+endpoint = "https://bldr.habitat.sh"
+```
+
+Then, issue the following command:
+```
+hab config apply builder-api.default $(date +%s) upstream.toml
+```
+After the config is successfully applied, the services should be configured to use the upstream.
+
```

Comment:
May be worth also adding a note: Alternatively, the config can be directly added to the builder-api user.toml by modifying the support/config.sh file.